ui/web/index.html

```diff
@@ -3,10 +3,316 @@
+                            <option value="6">Task 7</option>
+                            <option value="7">Task 8</option>
+                            <option value="8">Task 9</option>
+                            <option value="9">Task 10</option>
+                            <option value="10">Task 11</option>
+                            <option value="11">Task 12</option>            
+                        </select>                       
+                    </td>
+                    <td>
+                        <input type="button" value="Print" id="loadPrint">
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <div id="chartdiv" style="height:500px;width:300px; "></div>
+                    </td>
+                </tr>
+                </table>
+    </fieldset>
+
+    <script>  
+
+        var applicationContext = {};
+
+        var tasksObject = {};
+        var tasksPlot = {
+            tasksObject
+        };
+                                   
+        function loadTasks() {
+            var columns_mapping = {};
+            var requestBody = {
+            columns_mapping
+            };
+            var excelFile = $('#excelFile').get(0).files.item(0);
+            requestBody.sheet = $("#sheetName").val();
+            requestBody.first_row = +$("#firstRow").val();
+            requestBody.last_row = +$("#lastRow").val();
+            requestBody.rows_to_skip = $("#rowSkip").val();
+            columns_mapping.uid = $("#uid").val();
+            columns_mapping.name = $("#nam").val();
+            columns_mapping.blockers = $("#blockers").val();
+            columns_mapping.min_estimate = $("#minEst").val();
+            columns_mapping.normal_estimate = $("#norEst").val();
+            columns_mapping.max_estimate = $("#maxEst").val(); 
+            $.ajax({
+                type: 'PUT',
+                url: 'http://localhost:5000/api/excelWorkbook',
+                data: excelFile,
+                contentType: 'application/vnd.ms-excel',
+                processData: false,
+                success: function (data) {
+                    alert("Response received: " + JSON.stringify(data));
+                    requestBody.file = data.tempFileName;
+                    $.ajax({
+                        type: 'POST',
+                        url: 'http://localhost:5000/api/excelWorkbook/tasks',
+                        contentType: 'application/json',
+                        data: JSON.stringify(requestBody),
+                        success: function (data) {
+                            alert("Response received: " + JSON.stringify(data));
+                            applicationContext.tasks = data.tasks;
+                        }    
+                    });    
+                }
+             });                           
+        };
+
+        $("#loadTasksButton").click(loadTasks); 
```

Comment:
It would be better to move this line to the bottom of this script. So we will have next structure of the script:

```
var applicationContext = {};

function loadTasks() {
  ...
}

function printTaskGraphic() {
  ...
}

$("#loadTasksButton").click(loadTasks); 
$("#loadPrint").click(printTaskGraphic);
```

Reply:
Line moved